src/Cli/dotnet/commands/dotnet-workload/install/FileBasedInstaller.cs

```diff
@@ -32,6 +32,7 @@ internal class FileBasedInstaller : IInstaller
         private readonly FileBasedInstallationRecordRepository _installationRecordRepository;
         private readonly PackageSourceLocation _packageSourceLocation;
         private readonly RestoreActionConfig _restoreActionConfig;
+        private string workloadSetPath;
```

Comment:
The convention is for private fields to start with an underscore.  That said, I don't think we will need this field at all if we get rid of the RollBackWorkloadSetInstallation method.